src/parser/jobs/dnc/modules/CommonData.tsx

```diff
@@ -0,0 +1,34 @@
+import ACTIONS from 'data/ACTIONS'
```

Comment:
not a module; would be good to move it outside the modules folder

Reply:
np, that's a fair comment. moved it up a folder level